cppapi/client/zmqeventconsumer.cpp

```diff
@@ -50,6 +50,9 @@
 #include <sys/types.h>
 #include <fcntl.h>
 #include <arpa/inet.h>
+#if defined(__OpenBSD__) || defined(__FreeBSD__)
+#include <netinet/in.h>
+#endif
```

Comment:
`<netinet/in.h>` is a standard POSIX header: https://pubs.opengroup.org/onlinepubs/009695399/basedefs/netinet/in.h.html

In fact, `<arpa/inet.h>` includes `<netinet/in.h>` (at least on Debian and Centos). We should simply drop the ifdef as it won't do any harm to include it twice.

```suggestion
#include <arpa/inet.h>
#include <netinet/in.h> // Needed for systems that do not include this file from arpa/inet.h
```